### ISSUE TITLE
Add list/matrix of SBML supporting software

### ISSUE THREAD

**Issue by matthiaskoenig** [sbmlteam/sbml-org-website#37]
There is no information on SBML supporting software. The software matrix before was not ideal, but it would be great if we can have a list somewhere. With the transition to a github repository this could be just a yaml file which people can easily update.
This list does not have to be perfect, but just give a starting point.

**Comment by Lyymrzy** [comment]
Yes, you are right. The software matrix before is not here.We should make the SBML_Software_Guide great again!
![图片](https://github.com/sbmlteam/sbml-org-website/assets/133517217/52ac2507-35c9-4770-a886-12b3d4c35700)
This is the picture in the teacher's courseware, but we can't access this website anymore. I hope it can repeat its glory!

**Comment by fbergmann** [comment]
The trouble with the old system was, that it was hard to keep the data up-to-date. And many of the tools in the list were sooner or later not available / updated anymore. My hope would be, that we could use other resources for this, maybe bio.tools would be a good alternative. 

<https://bio.tools/t?q=sbml>

If someone had time, they could query bio.tools for generating a display on the SBML website.
